app/controllers/ChangeSuccessController.scala

```diff
@@ -80,17 +80,16 @@ class ChangeSuccessController @Inject()(vatSubscriptionService: VatSubscriptionS
           "Rendering InternalServerError")
         authComps.errorHandler.showInternalServerError
       } { title =>
-        val viewModel = ChangeSuccessViewModel(title, user.session.get(verifiedAgentEmail), entityName, None)
+        val viewModel = ChangeSuccessViewModel(title, user.session.get(verifiedAgentEmail), entityName)
         Ok(tradingNameSuccessView(viewModel, isRemoval))
       }
     }
 
   private[controllers] def renderBusinessNameView(implicit user: User[_]): Future[Result] =
     vatSubscriptionService.getCustomerInfo(user.vrn).map { vatSubscriptionResponse =>
```

Comment:
Can we remove the call to vat subscription? I don't see why we need it now if we don't play back the business name or determine contact pref anymore. Same for the trading name function above. They look like unnecessary API calls to me.